src/dbt_score/cli.py

```diff
@@ -87,8 +87,8 @@ def cli() -> None:
     default=None,
 )
 @click.option(
-    "--fail_any_model_under",
-    help="Fail if any model is under this value.",
+    "--fail_any_evaluable_under",
+    help="Fail if any evaluable is under this value.",
```

Comment:
@jochemvandooren 

Were you okay with this change in configuration?  I think this would be my preference (a single setting to fail any item under a threshold), but I could see the argument for separate settings for models or sources as well.

Reply:
While there, could we use dashes `-` instead of underscores `_` in the CLI option? This was a mistake when introduced 🙏

Reply:
"evaluable" seems very correct and fancy, but perhaps not the best for the CLI. Maybe "entity" could work better, as in dbt entities are models, sources, etc. WDYT?

Reply:
updated